packages/console/src/components/MultiTextInput/utils.ts

```diff
@@ -10,12 +8,14 @@ export const validate = (
     return;
   }
 
-  if (!value || value.length === 0 || value.filter(Boolean).length === 0) {
-    return conditional(
-      rule.required && {
-        required: rule.required,
-      }
-    );
+  if (!value?.filter(Boolean).length) {
+    if (!rule.required) {
+      return;
+    }
+
+    return {
+      required: rule.required,
+    };
```

Comment:
I think it's equivalent to
```typescript
return { required: rule.required }
```

Reply:
No, the first one returns undefined, and the second one returns an object.